Development

ash1969 opened the pull request:
Profile Added
Display of Recent and Popular Tags

pkkp86nitd committed:
add question view done

visibility attribute added in question model

ash1969 committed:
#4 Views for Forum Page
#5 Views For Display Questions Page

pkkp86nitd committed:
create dislplay view updated

forum page done..

pkkp86nitd committed:
Merge branch 'development' into development

pkkp86nitd committed:
Merge pull request #1 from ash1969/development

Issues #4 and #5 resolved

pkkp86nitd committed:
update bug fixed

pkkp86nitd committed:
tags and question linked

pkkp86nitd committed:
Update db.sqlite3

pkkp86nitd committed:
Merge branch 'development' of https://github.com/pkkp86nitd/RECursionNITD-website into development

pkkp86nitd committed:
Update db.sqlite3

pkkp86nitd committed:
tags and question linked in tagging model

tagging model created and working

pkkp86nitd committed:
tag linked with question udation done

pkkp86nitd committed:
create and update tags working properly

ash1969 committed:
Changes Made

ash1969 committed:
Changes Made

ash1969 committed:
Changes Made

ash1969 committed:
Changes Made

ash1969 committed:
Changes Made

ash1969 committed:
Changes Made

ash1969 committed:
Changes Made

ash1969 committed:
Changes Made

ash1969 committed:
Changes Made

ash1969 committed:
Deletions

ash1969 committed:
Deletions

ash1969 committed:
Deletions

pkkp86nitd committed:
changes

ash1969 committed:
'Update' Modified

ash1969 committed:
Gave a try with formsets!

ash1969 committed:
Adding Questions Using Formsets!

ash1969 committed:
Bug Related To Formset Fixed!

ash1969 committed:
'Update' Questions Through Formsets

ash1969 committed:
Template Apppropriate For Formsets

ash1969 committed:
Deleted Unneccessary Print Statements

ash1969 committed:
Simplified View For Admin Site

ash1969 committed:
Optimized 'Update' Through Bulk Create

pkkp86nitd committed:
optimisation of update form

bulk addition of tags

ash1969 committed:
Creating Questions Through Bulk Create

pkkp86nitd committed:
create questions through bulk create

pkkp86nitd committed:
udating gitignore

db.sqlite3

pkkp86nitd committed:
urls without regexp

ash1969 committed:
Gave a try with formsets!

ash1969 committed:
Views Added For 'Following' Feature

ash1969 committed:
URL Added For 'Following' Feature

ash1969 committed:
Automatic Timestamp For Follows

ash1969 committed:
Template Edited For Display Of Follow Link

ash1969 committed:
Added Comments' Feature
Create and Update

ash1969 committed:
Template For Comment Feature

ash1969 committed:
URLS Added For Comment

ash1969 committed:
Added Comment Form

ash1969 committed:
Added Views For Comments

ash1969 committed:
Template Edited For Display Of Comment Link

pkkp86nitd committed:
answer add and update done

one user can give only 1 answer and questionnare cant answer  and answerer and update his/her answer

pkkp86nitd committed:
update db.sqlite

pkkp86nitd committed:
Revert "Template Edited For Display Of Comment Link"

This reverts commit 25429f78507df1c978f59bf110311060df8eb3de.

pkkp86nitd committed:
Revert "Revert "Template Edited For Display Of Comment Link""

This reverts commit 760b3500347e55be18a421763c9a86d5381fa699.

pkkp86nitd committed:
Revert "Revert "Revert "Template Edited For Display Of Comment Link"""

This reverts commit f9a118856656f2a481e38fb2bd20da7ea81cb4de.

pkkp86nitd committed:
Revert "Revert "Template Edited For Display Of Comment Link""

This reverts commit 760b3500347e55be18a421763c9a86d5381fa699.

pkkp86nitd committed:
Merge branch 'pr/2' into development

pkkp86nitd committed:
changes in pr for merging in developement

pkkp86nitd committed:
changes in pr

pkkp86nitd committed:
urls.py updated in pr

pkkp86nitd committed:
Merge branch 'pr/2' into development

pkkp86nitd committed:
comment and follow functions working properly after merging

pkkp86nitd committed:
redirecing again to details page after commenting or follow/Unfollow

earlier it was redircting to list page

pkkp86nitd committed:
upvote/Downvote function added

answerer cant use this feature for  own answer..logged in user can only do voting..

pkkp86nitd committed:
ordering of list questions on forum page by created_at date

pkkp86nitd committed:
logged out or anonymous user can access details of question

anonymous users can access details page but they need to login for  using any further feature

ash1969 committed:
Logged Out Users Can See Details

pkkp86nitd committed:
update db.sqlite

pkkp86nitd committed:
update db

pkkp86nitd committed:
Merge branch 'pr/3' into development

ash1969 committed:
Merge branch 'pr/2' into development

ash1969 committed:
Merge remote-tracking branch 'origin/development' into development

# Conflicts:
#	website/recursion_website/admin.py
#	website/recursion_website/forms.py
#	website/recursion_website/templates/comment.html
#	website/recursion_website/urls.py
#	website/recursion_website/views.py

ash1969 committed:
Unnecessary Print Statements Deleted

ash1969 committed:
Answer HTML Added After Syncing

ash1969 committed:
View Profile Feature Added

ash1969 committed:
Create Profile Feature Added!

ash1969 committed:
Update Profile Feature Added!

ash1969 committed:
Total Questions And Answers Displayed On Forum!

ash1969 committed:
Recent Tags Displayed

ash1969 committed:
Popular Tags Displayed

ash1969 committed:
Bugs Removed

ash1969 committed:
Recent and Popular Tags' Links Made Functional

ash1969 committed:
Bug Fixed

ash1969 committed:
Recent Tags Changed From Recently Entered To Recently Used

vjs22334 reviewed (changes requested):
1. use django signals system to create a post_save signal handler to create a empty profile when user registers.
2. after registration redirect users to create profile
3. add proper validation to the profile form (esp email and image url field)
4. add links to access the update and edit profile pages

pkkp86nitd committed:
question and answer description shoubld be of minimum length error display

question and answer description shoubld be of minimum length error display handled in views.py

pkkp86nitd committed:
pr4

pkkp86nitd committed:
env deleted

pkkp86nitd committed:
Merge branch 'pr/4' into development

ash1969 committed:
#After Registration Users Redirected To Create Profile
#No Id Required For Links To Create And Update Profile

pkkp86nitd committed:
question.html updated

ash1969 committed:
Model Altered To Check For Valid Email Id

ash1969 committed:
Django Signals System,Create An Empty Profile On Registration

ash1969 committed:
Checking For Too Short Answers And Questions!

ash1969 committed:
Links To Users' Profile In Detail Question Page Added Along With View My Profile Link

ash1969 committed:
Choice Field Added For Role And 'User' Assigned By Default

pkkp86nitd committed:
Events CRUD

CUD only for admins

pkkp86nitd committed:
one user can answer only once

after ansewering once  answer should only be updated...flaws fixed in html as well in views

ash1969 committed:
Custom Validation For Image Url

ash1969 committed:
Downloading Image From Remote Server To Local Server And Using Relative Paths To Directory

ash1969 committed:
Getting The Path of Directory Of The Settings File And Setting It As Base Path

pkkp86nitd committed:
events section updated

events app CRUD done

pkkp86nitd committed:
questions.html added

to remove merge conflict

ash1969 committed:
Removing Merge Conflict

ash1969 committed:
questions.html Added

pkkp86nitd committed:
db.sqlite update

pkkp86nitd committed:
pr5 update

pkkp86nitd committed:
Merge branch 'development' into pr/5

pkkp86nitd committed:
merged

ash1969 committed:
Merge pull request #7 from pkkp86nitd/pr/5

Merging Events' Section

ash1969 committed:
Downloading Images For Events

ash1969 committed:
Comments On Answer Added

ash1969 committed:
Saving Every Image With .png Extension

ash1969 committed:
Importing Validators

ash1969 committed:
Added Email Confirmation During User Registration

ash1969 committed:
Added Change Password Option

ash1969 committed:
Added Password Reset Through Email Feature

ash1969 committed:
New user_profile App Created!

ash1969 committed:
Notifying Moderators Through Email Upon Activity In AskREC

ash1969 committed:
Not Blocking User Account On Submitting Email, Password Reset

ash1969 committed:
Notifying Followers Through Email Upon Activity On Question

ash1969 committed:
Sending Email To A Moderator+Follower Only Once

ash1969 committed:
Submitting user_profile Forms With Ajax+jQuery

ash1969 committed:
Tags Displayed For Question On Detail Page

vishvanath45 committed:
Add markdown support

ash1969 committed:
Merge pull request #8 from RECursion-NITD/md_support

Add Markdown Support
